lib/plugin-loader.js

```diff
@@ -858,10 +858,10 @@ PluginLoader.prototype = {
           count: newPlugins.length
         });
       }
-  },
+  }),
 
   // Note - Not to be confused with auth capabilities, that describe what an auth plugin can do
-  async getComponentCapabilities(config) {
+  getComponentCapabilities: Promise.coroutine(function*(config) {
```

Comment:
Why it is a `coroutine` if it returns a Promise? A coroutine can `yield` a promise.